src/modules/dashboard/components/FeaturedChannelListItem.tsx

```diff
@@ -0,0 +1,223 @@
+// React + Web3 Essentials
+import { FC, useMemo, useState, useEffect } from 'react';
+
+// External Packages
+import { useSelector } from 'react-redux';
+import { cloneDeep } from 'lodash';
+
+// Internal Components
+import { Box, Button, CaretDown, NotificationMobile, Skeleton, Text } from 'blocks';
+import SubscribeChannelDropdown from 'common/components/SubscribeChannelDropdown';
+import UnsubscribeChannelDropdown from 'common/components/UnsubscribeChannelDropdown';
+
+// Internal Configs
+import { ChannelDetailsProps } from '../configs/DashboardFeaturedChannels.config';
+import { useGetChannelDetails } from 'queries';
+import { css } from 'styled-components';
+import Ethereum from 'blocks/illustrations/components/Ethereum';
+import { LOGO_ALIAS_CHAIN } from '../configs/ChainDetails';
+import TickDecoratedCircleFilled from 'blocks/icons/components/TickDecoratedCircleFilled';
+import { Link } from 'react-router-dom';
+import VerifiedToolTipComponent from './VerifiedToolTipComponent';
+
+type FeaturedChannelListItemProps = {
+  channel: ChannelDetailsProps;
+};
+
+const FeaturedChannelListItem: FC<FeaturedChannelListItemProps> = (props) => {
```

Comment:
```suggestion
const FeaturedChannelsListItem: FC<FeaturedChannelsListItemProps> = (props) => {
```

@abhishek-01k also change th file name